src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -194,17 +196,14 @@ public double getWristPosRelativeToGround() {
         return getArmPos() + wristEncoder.getPosition();
     }
 
-    public void setArmTarget(double target) {
-        goalPosRad[ARM] = getArmClampedGoal(target);
-    }
-
-    public void setWristTarget(double target) {
-        goalPosRad[WRIST] = getWristClampedGoal(target);
+    public void setArmTarget(double targetPos, double targetVel) {
+        goalState[ARM].position = getArmClampedGoal(targetPos);
+        goalState[ARM].velocity = MathUtil.clamp(targetVel, 0, MAX_FF_VEL[ARM]);
```

Comment:
This is wrong and not needed. Wrong because the goal velocity could be negative and not needed because the TrapezoidProfile velocity limit will take care of it.